lib/uniffi-zcash/src/udl/zcash_primitives/common.udl

```diff
@@ -0,0 +1,9 @@
+dictionary ZcashDiversifierIndexAndPaymentAddress {
+  ZcashDiversifierIndex diversifier_index;
+  ZcashPaymentAddress address;
```

Comment:
I think `ZcashPaymentAddress` belongs to the `sapling` package. So im unsure about exposing here/reusing this type 🤔

From my POV, it represents the output of a concrete method of an object (a workaround for not having tuples). If this gets reused in another object/method, and a breaking change its needed by any other object/method, that could affect to all the APis that made use of this unified result struct. 

Would it be better to just copy the type in case we need a similar workaround ?

Reply:
In the other hand, i see it looks a well stablished pattern in the zcash API. Naming the dictionary with the types it contains would limit its use. In case of breaking change, a new one would needed and thats should be enough.

Reply:
It's just workaround for not being able to return pairs (tuples).

Reply:
Agreed, ready with this one :)